sqlglot/expressions.py

```diff
@@ -6054,6 +6054,10 @@ class UtcDate(Func):
     arg_types = {}
 
 
+class UtcTime(Func):
+    arg_types = {}
```

Comment:
I'm assuming you investigated similarly whether other dialects support `UTC_TIME`, right?

Reply:
Sounds good, thanks!